.github/workflows/linux.yml

```diff
@@ -113,6 +113,9 @@ jobs:
       - name: Build PGroonga by Meson
         run: |
           meson compile -C pgroonga.build
+      - name: Install PGroonga by Meson
+        run: |
+          sudo meson install -C pgroonga.build
```

Comment:
Can we remove `sudo` by specifying `--prefix` in `meson setup`?

Reply:
Then we should use `meson install --destdir` without `sudo`.
We should not mix artifacts built by Meson and Makefile.